nilearn/datasets/atlas.py

```diff
@@ -185,8 +186,8 @@ def fetch_atlas_craddock_2012(atlas_name=None, number_of_regions=None,
     atlas_name_file = dict(zip(atlas_items, sub_files))
 
     # select indicated atlas file
```

Comment:
This older comment is unnecessary noise, your code explains what's happening very nicely.